packages/core/src/forms/forms.stories.ts

```diff
@@ -37,7 +37,7 @@ export const nativeHTML5ValidationSingle = () => {
   return html`
     <cds-input validate>
       <label>text input (required)</label>
-      <input placeholder="placeholder text" required />
+      <input placeholder="place holder text" required />
```

Comment:
Screen readers now read this as `PLACE holder`.

Previously, it was being read as `PLASS holder`